websites/api-docs-smoke-test/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools-website/api-docs-smoke-test",
-  "version": "3.0.18",
+  "version": "1.0.0",
```

Comment:
Why changing the versions? Those are managed by changesets

Reply:
> Why changing the versions? Those are managed by changesets

From one of my discussions with @nkuehn I understand we don't do releases for test sites so that's why I removed everything associated with that.

Reply:
Yes but it's part of the monorepo/workspace and so changeset will include that as well. It's not a problem in general, those packages are not published anyway.

Revert those changes, as otherwise changeset will regenerate them again next time.